build.sbt

```diff
@@ -1806,9 +1808,11 @@ lazy val `bench-libs` = (project in file("std-bits/benchmarks"))
         (runtimeClasspath ++ runtimeInstrumentsClasspath)
           .map(_.data)
           .mkString(File.pathSeparator)
+      // Only run ServiceProvider processor and ignore those defined in META-INF, thus
+      // fixing incremental compilation setup
```

Comment:
This comment is copy left. I don't think we restrict annotation processors in this place anymore.